### PR TITLE
{lib}[gompic/2020a] Boost.Python v1.72.0 w/ Python 2.7.18 + 3.8.2

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso

File: easybuild/easyconfigs/b/Boost.Python/Boost.Python-1.72.0-gompic-2020a.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+version = '1.72.0'
+
+homepage = 'https://boostorg.github.io/python'
+description = """Boost.Python is a C++ library which enables seamless interoperability between C++
+ and the Python programming language."""
+
+toolchain = {'name': 'gompic', 'version': '2020a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://dl.bintray.com/boostorg/release/%(version)s/source/']
```
Comment (branfosj) [inline easybuild/easyconfigs/b/Boost.Python/Boost.Python-1.72.0-gompic-2020a.eb:13]:
```suggestion
source_urls = ['https://boostorg.jfrog.io/artifactory/main/release/%(version)s/source/']
```

File: easybuild/easyconfigs/b/Boost/Boost-1.72.0-gompic-2020a.eb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name = 'Boost'
+version = '1.72.0'
+
+homepage = 'https://www.boost.org/'
+description = """Boost provides free peer-reviewed portable C++ source libraries."""
+
+toolchain = {'name': 'gompic', 'version': '2020a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://dl.bintray.com/boostorg/release/%(version)s/source/']
```
Comment (branfosj) [inline easybuild/easyconfigs/b/Boost/Boost-1.72.0-gompic-2020a.eb:10]:
```suggestion
source_urls = ['https://boostorg.jfrog.io/artifactory/main/release/%(version)s/source/']
```

**Review by branfosj — CHANGES_REQUESTED** [review]
bintray is closing, so the `source_urls` need updating. See #12099 for more information.

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bber0501u03a.bb2.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/3a4ff6fa809c8c6ccb784f5b8f18b5c5 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @akesandgren!

**Pull request merged**